src/index.html

```diff
@@ -1,12 +1,342 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
it's better to left `html` tag without class

Reply:
understand , thanks